cli/input/readpass_unix.go

```diff
@@ -0,0 +1,30 @@
+//go:build !windows
+// +build !windows
```

Comment:
Starting from `go1.17` the first version is preferred, could you add an issue for that?

Reply:
Sure, https://github.com/nspcc-dev/neo-go/issues/2497.